src/adaptors/null-articles.ts

```diff
@@ -0,0 +1,35 @@
+import { JsonLdObj, Iri } from 'jsonld/jsonld-spec';
+import Articles from './articles';
+
+export default class NullArticles implements Articles {
```

Comment:
Many lint errors are introduced by this class:

```
/app/src/adaptors/null-articles.ts
   5:20  error    Expected 'this' to be used by class method                 class-methods-use-this
  16:14  error    Expected 'this' to be used by class async method 'count'   class-methods-use-this
  20:12  error    Expected 'this' to be used by class async method 'get'     class-methods-use-this
  24:12  error    Expected 'this' to be used by class async method 'set'     class-methods-use-this
  24:13  warning  'node' is defined but never used                           @typescript-eslint/no-unused-vars
  28:15  error    Expected 'this' to be used by class async method 'delete'  class-methods-use-this
  28:16  warning  'id' is defined but never used                             @typescript-eslint/no-unused-vars
  28:40  error    Unexpected empty async method 'delete'                     @typescript-eslint/no-empty-function
  32:12  error    Expected 'this' to be used by class async method 'has'     class-methods-use-this
  32:13  warning  'id' is defined but never used                             @typescript-eslint/no-unused-vars
```

Reply:
A lot of these don't sound right, but maybe could just use `InMemoryArticles` and don't add anything to it for now.

Reply:
Resolving as we have removed the NullArticles class for now.